utils/writers.ts

```diff
@@ -0,0 +1,90 @@
+import { flow, pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as E from "fp-ts/lib/Either";
+import { upsertBlobFromText } from "@pagopa/io-functions-commons/dist/src/utils/azure_storage";
+import { BlobService } from "azure-storage";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import {
+  AssertionFileName,
+  LolliPOPKeysModel,
+  NewLolliPopPubKeys,
+  RetrievedLolliPopPubKeys
+} from "../model/lollipop_keys";
+import {
+  cosmosErrorsToString,
+  ErrorKind,
+  InternalError,
+  toInternalError
+} from "./errors";
+
+export type PopDocumentWriter = (
+  item: NewLolliPopPubKeys
+) => TE.TaskEither<InternalError, RetrievedLolliPopPubKeys>;
+
+export type AssertionWriter = (
+  assertionFileName: AssertionFileName,
+  assertion: string
+) => TE.TaskEither<InternalError, true>;
+
+// IMPLEMENTATION
+export const getPopDocumentWriter = (
+  lollipopKeysModel: LolliPOPKeysModel
+): PopDocumentWriter => (item): ReturnType<PopDocumentWriter> =>
+  pipe(
+    lollipopKeysModel.upsert(item),
+    TE.mapLeft(error => ({
+      detail: cosmosErrorsToString(error),
+      kind: ErrorKind.Internal as const
+    }))
+  );
+
+export const getAssertionWriter = (
+  assertionBlobService: BlobService,
+  lollipopAssertionStorageContainerName: NonEmptyString
+): AssertionWriter => (
+  assertionFileName,
+  assertion
+): ReturnType<AssertionWriter> =>
+  pipe(
+    TE.tryCatch(
+      () =>
+        new Promise<boolean>((resolve, reject) =>
+          assertionBlobService.doesBlobExist(
```

Comment:
can we taskify this?

Reply:
done in https://github.com/pagopa/io-functions-lollipop/pull/13/commits/d1b5e1f7cbb0852a16b31f9b7f272591db593b35